src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1233,6 +1452,184 @@ input ParkingFilter {
     select: [ParkingFilterOperation!]
 }
 
+"""
+Settings that control the behavior of itinerary filtering. These settings are meant mainly
+for debugging purposes.
+"""
+input PlanItineraryFilterSettingsInput {
```

Comment:
I copied some fields from the transmodel API for this but I didn't copy `transitGeneralizedCostLimit` as I didn't understand it's documentation completely and not sure if we should actually have a DoubleFunction scalar for it or also somehow split that into an input type with two fields?